src/domain/services/DeleteLostAndTemporaryFilesService.ts

```diff
@@ -20,28 +20,40 @@ export class DeleteLostAndTemporaryFilesService {
      * - return in getStorage() method object that implements IFileStorage interface
      */
     async deleteLostAndTemporaryFiles(storageName: FileStorage): Promise<void> {
+        const lostAndTemporaryFilesPaths = await this.getLostAndTemporaryFilesPaths(storageName);
+        const storage = this.getStorage(storageName);
+        if (storage) {
+            lostAndTemporaryFilesPaths.forEach(storage.deleteFile);
+        }
```

Comment:
Если нет storage, то бессмысленно искать файлы:
```suggestion
        const storage = this.getStorage(storageName);
        if (!storage) {
            return;
        }
        const lostAndTemporaryFilesPaths = await this.getLostAndTemporaryFilesPaths(storageName);
        lostAndTemporaryFilesPaths.forEach(storage.deleteFile);
```

И еще вопрос - а storage.deleteFile это синхронная операция? Не нужно ее ждать?

Reply:
>И еще вопрос - а storage.deleteFile это синхронная операция? Не нужно ее ждать?

Бывает всякой =)

```ts
deleteFile(fileName: string): void | Promise<void>;
```

Reply:
Так в команде через await ожидается результат работы юзкейса. Если storage.deleteFile окажется асинхронным, то ожидание в команде будет лишено смысла (удаление будет происходить после того как напишется "удаление завершено")

Reply:
Да, я поэтому поставил await

Reply:
Куда?

Reply:
Я снова ревью запросил, ты, наверное, не видел ещё

https://github.com/steroids/nest-file/pull/14/files#diff-21b0db0e23f02b4433538d1a77a7491c79d10e5d2097ee81b704f7525ab48e14